notebooks/en/mlflow_ray_serve.ipynb

```diff
@@ -0,0 +1,1079 @@
+{
```

Comment:
<p>Let's add the following exercises as well:</p><ul><li>Handle optional values in the MLflow model signature</li><li>Handle <a href="https://mlflow.org/docs/latest/model/signatures.html#model-signatures-with-inference-params" target="_blank">inference parameters</a></li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/258/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>